overlord/hookstate/ctlcmd/export_test.go

```diff
@@ -48,6 +48,12 @@ func MockDevicestateSystemModeInfoFromState(f func(*state.State) (*devicestate.S
 	return func() { devicestateSystemModeInfoFromState = old }
 }
 
+func MockHasSnapdControlInterface(f func(st *state.State, snapName string) (bool, error)) (restore func()) {
+	old := hasSnapdControlInterface
+	hasSnapdControlInterface = f
+	return func() { hasSnapdControlInterface = old }
```

Comment:
nitpick: you could use testutil.Backup()

Reply:
Fixed! Thanks for spotting that one - I already knew about testutil.Backup but it must have slipped my mind there!